ui/admin/app/abilities/credential.js

```diff
@@ -6,7 +6,8 @@ export default class OverrideCredentialAbility extends CredentialAbility {
   @service features;
 
   get canRead() {
-    return (this.features.isEnabled('json-credentials') || this.model.type !== 'json')
+    return this.features.isEnabled('json-credentials') ||
+      this.model.type !== 'json'
```

Comment:
Not blocking but I wanted to give context here. This was one lined for better readability, along with the explicit `()`. Moving the `this.model.type ...` portion to the next line had caused confusion.

Reply:
It might exceed the line length of 80?  This happened after running `yarn format`.

Reply:
Yep it sure does exceed 80. Do you think the readability of this is crucial enough that it deserves a rule exception? I assume that would block the format from changing it as well.

Reply:
Hmm Actually there may be a more concise rewrite!  Let me propose something.